query/src/test/java/org/infinispan/query/config/DeclarativeAutoConfigTest.java

```diff
@@ -41,7 +41,7 @@ public void call() {
             assertEquals(properties.getProperty("hibernate.search.default.exclusive_index_use"), "true");
             assertEquals(properties.getProperty("hibernate.search.default.reader.strategy"), "shared");
             assertEquals(properties.getProperty("hibernate.search.default.indexmanager"), "near-real-time");
-            assertEquals(properties.getProperty("hibernate.search.backends.infinispan_backend.directory.type"), "filesystem");
+            assertEquals(properties.getProperty("hibernate.search.backends.infinispan_backend.directory.type"), "local-filesystem");
```

Comment:
could we put these strings into constants ?

Reply:
There is a plan of revamping the indexing config, that will not be string based anymore, so those strings are going to be hidden https://issues.redhat.com/browse/ISPN-11797

Reply:
ok